prometheus-exporters/openstack-exporter/values.yaml

```diff
@@ -8,9 +8,8 @@ exporter:
   log_level: "DEBUG"
 openstack:
   enabled: false
-  image:
-    name: openstack-exporter
-    tag: DEFINED-IN-REGION-SECRETS
+  imageName: openstack-exporter
+  imageVersion: DEFINED-IN-REGION-SECRETS
```

Comment:
Can we at the same time get rid of this anti-pattern? `DEFINED-IN-REGION-SECRETS` is not a usable default!

Reply:
I like it, but it makes the CI fail. We need to add a `ci/` directory with some test values now that we don't have (non-working) defaults.
```
engine.go:159: [INFO] Missing required value: .Values.global.openstack_exporter_master_password missing
engine.go:159: [INFO] Missing required value: .Values.global.openstack_exporter_master_password missing
engine.go:159: [INFO] Missing required value: .Values.global.region missing
engine.go:159: [INFO] Missing required value: .Values.openstack.alerts.prometheus missing
engine.go:159: [INFO] Missing required value: .Values.global.registry missing
engine.go:159: [INFO] Missing required value: .Values.openstack.imageVersion missing
Error: 1 chart(s) linted, 1 chart(s) failed
```